src/views/LoadManagementConfig.vue

```diff
@@ -70,6 +70,39 @@
 							</p>
 						</template>
 					</openwb-base-button-group-input>
+					<openwb-base-select-input
+						title="Hausverbrauch"
+						:options="getHcSourceIdOptions()"
```

Comment:
Für die Aufteilung in `options` und `groups`:
```suggestion
						:options="getHcSourceIdOptions.options"
						:groups="getHcSourceIdOptions.groups"
```
Die Vue-Komponente erzeugt dann eine Liste, in der die `options` vor den `groups` stehen.